lib/internal/bootstrap_node.js

```diff
@@ -486,8 +486,9 @@
   };
 
   NativeModule.wrapper = [
-    '(function (exports, require, module, __filename, __dirname, process, global) { ',
-    '\n});'
+    '(function (exports, require, module, __filename, __dirname, process, global) { ' +
+    'return function (exports, require, module, __filename, __dirname) { ',
```

Comment:
@ide is the `return` statement needed here?

Looks like return value for this function is unused when it is called on line 506